website/content/api-docs/secret/databases/index.mdx

```diff
@@ -534,6 +534,11 @@ this in order to know the password.
 - `username` `(string: <required>)` – Specifies the database username that this
   Vault role corresponds to.
 
+- `self_managed_password` `(string)` – <EnterpriseAlert product="vault" inline />
+  The password corresponding to this username in the database. Required when using
+  the Rootless Password Rotation workflow for static roles. Only enabled for select
+  DB engines.
```

Comment:
```suggestion
  the Rootless Password Rotation workflow for static roles. Only available for select
  DB engines. See the [database secrets engine capabilities](/vault/docs/secrets/databases#database-capabilities) for details.
```
I think we should update the [DB capabilities](https://developer.hashicorp.com/vault/docs/secrets/databases#database-capabilities) table with the support for rootless static role rotation